addons/l10n_pk/demo/account_demo.py

```diff
@@ -0,0 +1,382 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+from datetime import datetime
+
+from odoo import api, models, Command
+from odoo.exceptions import UserError, ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountChartTemplate(models.AbstractModel):
+    _inherit = "account.chart.template"
+
+    @api.model
+    def _get_demo_data(self, company=False):
+        if company.account_fiscal_country_id.code == "PK":
+            if company.state_id:
+                return {
+                    'res.partner': self._get_demo_data_partner(company),
+                    'account.move': self._get_demo_data_move(company),
+                    'res.config.settings': self._get_demo_data_config_settings(company),
+                }
+            else:
+                _logger.error(
+                    'Error while loading Accounting(Pakistan) demo data in the company "%s".',
+                    company.name
+                )
+        else:
+            return super()._get_demo_data(company)
+
+    @api.model
+    def _get_demo_data_partner(self, company=False):
+        company = company or self.env.company
+        state_id = self.env['res.country.state'].search([
+            ('code', '=', 'IS/ICT'),
+            ('country_id', '=', company.country_id.id)
+        ], limit=1)
+        default_partner_dict = {
+            'company_id': company.id,
+            'country_id': 'base.pk',
```

Comment:
```suggestion
            'country_id': self.env.ref('base.pk').id,
```
Is this thing even working?

Reply:
Yes, both are functional, but the approach you mentioned is more preferable 👍